util/dvsim/SimCfg.py

```diff
@@ -108,6 +108,11 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
             self.en_build_modes.append("xprop")
         if self.build_seed:
             self.en_build_modes.append("build_seed")
+        else:
+            # TODO: when build_seed mode is not enabled, the script should not
+            # pass `--otp-seed` args. Temp support this by providing default
+            # build seed value.
+            self.build_seed = 10556718629619452145
```

Comment:
We may want to correct this in the argument parsing step of the individual generation scripts so that a non-numeric / empty flag defaults back to the value provided in Hjson. Can you file an issue so we do not forget to fix this?

Reply:
Sounds good! Here is the issue: https://github.com/lowRISC/opentitan/issues/12114